frontend/lit/TagReferences/Main.js

```diff
@@ -186,7 +189,13 @@ class TagReferencesMain extends Component {
                                 }}>
                                 &times;
                             </button>
-                            <div dangerouslySetInnerHTML={{__html: store.config.instructions}} />
+                            <div
+                                dangerouslySetInnerHTML={
+                                    store.config.instructions.length > 0
+                                        ? {__html: store.config.instructions}
+                                        : {__html: "No screening instructions found."}
```

Comment:
can we just hide the screening instructions button/div instead if none are available?

Reply:
good idea!! didn't think about hiding the button...